test/utils/nuxt.js

```diff
@@ -33,5 +33,14 @@ export const loadFixture = async function (fixture, overrides) {
   config.dev = false
   config.test = true
 
+  // disable terser to speed-up fixture builds
+  if (config.build) {
+    if (!config.build.hasOwnProperty('terser')) {
```

Comment:
isn't config.build.terser enough?

Reply:
You mean instead of `hasOwnProperty`? Probably yes, will change that